packages/front-end/components/EventWebHooks/utils.tsx

```diff
@@ -8,6 +8,12 @@ export type EventWebHookEditParams = {
   events: NotificationEventName[];
 };
 
+export const notificationEventNames = [
+  "feature.created",
+  "feature.updated",
+  "feature.deleted",
+] as const;
```

Comment:
Unfortunately I've needed to define these a second time. If I import the ones from `base-types.ts` in the backend package, I get a Webpack error (I think it can only import types at this time, and not values):

```
../back-end/src/events/base-types.ts
Module parse failed: Unexpected token (8:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   "feature.updated",
|   "feature.deleted",
> ] as const;
| 
| export type NotificationEventName = typeof notificationEventNames[number];
```

If anyone has any suggestions for how to deal with that so we don't have to duplicate this definition, that would be great, but for now I've just copy/pasted it. This will become inconvenient to keep in sync as we add more event names. The initial definition is [here](https://github.com/growthbook/growthbook/blob/main/packages/back-end/src/events/base-types.ts#L1-L8).

Reply:
I could be wrong on this one, so take it with a grain of salt. I believe to make this work, we need to:
1. Create a `.d.ts` file for these types in `packages/back-end/types/`
2. `yarn build` on the backend package so that the frontend can pick it up (`yarn dev` won't compile exportable types, I think...?)

Reply:
You cannot put non-types in a `.d.ts` file. Here is the error:

```
Error: Cannot find module '../events/base-types'
```

The thing I'm trying to access is not a type:

```ts
export const notificationEventNames = [
  "feature.created",
  "feature.updated",
  "feature.deleted",
] as const;
```

Reply:
For now I'm going to leave it as-is, with the duplication. If we find we need to duplicate more things like this, we can consider adding another package, e.g. `@growthbook/core` internal to the monorepo, where we can move our shared types and values.

Right now we can technically import any type from any file on the back-end even though it wasn't intended to be used externally in the package. What we can perhaps strive for in the future is to create a separate package, e.g. `@growthbook/core`, where we share types and values explicitly, and prevent ourselves from importing from other packages except `core` with eslint rules.